frontend/src/pages/Notifications/NotificationListItems.js

```diff
@@ -88,14 +101,15 @@ const NotificationListItems = ({
             secondary={createdAt}
           />
           <div className={classes.button}>
-            <Fab
-              size="small"
-              disabled={!isAllowedToSee(notification)}
-              color="primary"
-              onClick={() => history.push(redirectUri)}
-            >
-              <LaunchIcon />
-            </Fab>
+            {isAllowedToSee(notification) ? (
+              <Tooltip id="tooltip-inspect" title={strings.common.view}>
+                <div>
+                  <IconButton disabled={!isAllowedToSee(notification)} onClick={() => history.push(redirectUri)}>
```

Comment:
We already checked if the user is allowed to see the notification (see Line 104)

Reply:
removed since button is not displayed if user is not allowed to see the resource